zeroconf/__init__.py

```diff
@@ -852,7 +852,7 @@ def read_name(self) -> str:
         return result
 
 
-class DNSOutgoing:
+class DNSOutgoing(QuietLogger):
```

Comment:
I don't remember the details about `QuietLogger` precisely, but its methods are static so let's not inherit from it (I know some classes already do that – I'll remove it).

Reply:
Ok, I changed mine back but didn't change the others to avoid a merge conflict since it sounds like you're planning to fix those.